### PR TITLE
Copy emote link 3

### PR DESCRIPTION
**Pull request by sougataghar477**

### CODE REVIEW

File: src/components/utility/EmoteCard.vue [hunk 1]
```diff
@@ -253,6 +253,19 @@ const openContext = (ev: MouseEvent) => {
 
 			default:
 				break;
+			case "copy-emote-link":
+				navigator.clipboard.writeText(`https://7tv/emotes/${props.emote.id}`).then(() => {
```
Comment (AnatoleAM) [inline src/components/utility/EmoteCard.vue:257]: Don't hardcore the site url, use `window.location.origin` instead

File: src/components/utility/EmoteCard.vue [hunk 2]
```diff
@@ -253,6 +253,19 @@ const openContext = (ev: MouseEvent) => {
...
+					/* clipboard successfully set */
+					modal.open("copy-emote-link", {
```
Comment (AnatoleAM) [inline src/components/utility/EmoteCard.vue:259]: A modal would be slightly obnoxious just for copying. A tooltip or other minor form of feedback like a checkmark icon popping up next to the clicked button, would work better

**Review by AnatoleAM — CHANGES_REQUESTED** [review]

File: src/components/modal/CopiedEmoteLink.vue [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+</script>
+
+<style lang="scss" data-style-group="modal">
+@import "@scss/themes.scss";
+
+.copy-emote-link {
+	padding: 3em 0;
+	text-align: center;
+}
+</style>
```
Comment (AnatoleAM) [inline src/components/modal/CopiedEmoteLink.vue:23]: Remove deadcode

File: locale/en_US.ts [hunk 4]
```diff
@@ -157,6 +157,7 @@ export default {
 		add: "Add Emote",
 		author: "Added by",
 		created_at: "Created",
+		copied: "Emote link copied to clipboard",
```
Comment (AnatoleAM) [inline locale/en_US.ts:160]: This string is not used anywhere

**Review by AnatoleAM — CHANGES_REQUESTED** [review]

File: src/components/utility/EmoteCard.vue [hunk 5]
```diff
@@ -312,13 +314,16 @@ interface Indicator {
 			color: gray;
 			font-size: 0.65em;
 			margin-top: 0.1em;
+
```
Comment (AnatoleAM) [inline src/components/utility/EmoteCard.vue:317]: Why add whitespace here?

File: src/components/utility/EmoteCard.vue [hunk 6]
```diff
@@ -312,13 +314,16 @@ interface Indicator {
...
 			span.aka {
 				margin-right: 0.3em;
 			}
+
```
Comment (AnatoleAM) [inline src/components/utility/EmoteCard.vue:321]: Why add whitespace here?

File: src/components/utility/EmoteCard.vue [hunk 7]
```diff
@@ -312,13 +314,16 @@ interface Indicator {
...
 			span.og-name {
 				font-weight: 600;
 			}
 		}
+
```
Comment (AnatoleAM) [inline src/components/utility/EmoteCard.vue:326]: Why add whitespace here?

File: src/components/utility/EmoteCard.vue [hunk 8]
```diff
@@ -354,6 +359,7 @@ interface Indicator {
 				margin-top: 1em;
 			}
 		}
+
```
Comment (AnatoleAM) [inline src/components/utility/EmoteCard.vue:362]: Why add whitespace here?

File: src/components/utility/EmoteCard.vue (hunk 5 above)
Comment (sougataghar477) [reply src/components/utility/EmoteCard.vue:317]: its not me I swear , my eslint fucks up, lemme remove em

**Pull request closed without merging**